plugins/oauth/components/AuthenticatedClusterDropdown.js

```diff
@@ -0,0 +1,14 @@
+import ClusterDropdown from "#SRC/js/components/ClusterDropdown";
+
+class AuthenticatedClusterDropdown extends ClusterDropdown {
```

Comment:
is it okay to extend react components? AFAIK react doesnt really support this pattern 🤔